package.json

```diff
@@ -1,11 +1,10 @@
 {
   "name": "backpack",
-  "engines": {
-    "node": ">=6.0.0",
-    "npm": ">=4.0.0"
-  },
   "private": true,
   "scripts": {
+    "preinstall:node": "node --version | grep -q 8.6.0",
+    "preinstall:npm": "npm --version | grep -q 5.4.2",
```

Comment:
I'm assuming this will return exit code 1 i grep doesn't find the correct version, but will that stop the install?

Reply:
Just verified with `"preinstall": "false"` and yes it does halt

Reply:
Awesome, I wasn't 100% sure, thanks!